sql/_05_plcsql/_01_testspec/_01_basic_structure/_01_create_procedure/_12_comment/cases/01_comment.sql

```diff
@@ -0,0 +1,14 @@
+--+ server-message on
+
+create procedure comment_test() as
+begin
+    dbms_output.put_line('');
+end
+comment 'comment test';
+
+--select comment_test() as "col" from dual;
+select * from _db_stored_procedure where sp_name='comment_test';
+
```

Comment:
select sp_name, comment from _db_stored_procedure where sp_name='comment_test';

Reply:
I think, need to check sp_type, return_type, arg_count, and target too.
if that column is checked by another tc then I will reflect that.

Reply:
We are decision to use a '*'.